rust/pact_ffi/Cargo.toml

```diff
@@ -31,7 +31,8 @@ maplit = "1.0.2"
 multipart = { version = "0.18.0", default-features = false, features = ["client", "mock"] }
 onig = { version = "6.4.0", default-features = false }
 pact_matching = { version = "~1.2.2", path = "../pact_matching" }
-pact_mock_server = { version = "~1.2.6" }
+pact_mock_server = { version = "=1.2.8", git = "https://github.com/you54f/pact-core-mock-server.git", branch = "main" }
```

Comment:
please update to released pact_mock_server, when pact_matching is released, and pact_mock_server updated to reference released crate